package.json

```diff
@@ -13,16 +14,27 @@
     "select",
     "text"
   ],
-  "author": "Tom MacWright",
+  "author": "Mapbox",
   "license": "ISC",
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || ^15.0.0"
+    "prop-types": "^15.5.0",
+    "react": "^15.5.0"
```

Comment:
I don't think this number needs to be so restrictive, right? Could just as well stay as it was?